### PR TITLE
New topic on language nuances

### PR DESCRIPTION
**Pull request by LooseEthics**
Wanted to write a psychology topic, went off on a tangent and made this instead.
First actual contribution of mine, but hopefully it's up to the standard.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -14267,3 +14267,33 @@ label monika_hot_springs:
...
+        )
+    )
+
+label monika_languagenuances:
+    m 1eua "Hey [player], have you ever tried reading through a dictionary?"
+    m 3eua "Not necessarily because there was some word or expression you didn't know the meaning of, but just... {w=0.2}because?"
+    m 3lksdlb "Doesn't exactly sound like the most engaging of pastimes, I know."
+    m 3esb "But it can certainly be an interesting, even rewarding, way to spend some free time. Especially if it's a dictionary of a language you're still learning."
+    m 1esb "Many words have multiple meanings and, aside from the obvious benefits, knowing those can really help you see the finer points of the language."
+    m 1esa "Understanding these subleties can save you a lot of embarassment when you actually speak to someone."
```
Comment (Iniquitatis) [inline Monika After Story/game/script-topics.rpy:14289]: "embarrassment"

Comment (Rai99) [inline Monika After Story/game/script-topics.rpy:14289]:
```suggestion
    m 1esa "Understanding these subtleties can save you a lot of embarrassment when you actually speak to someone."
```

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -14267,3 +14267,33 @@ label monika_hot_springs:
     m 5dkbfb "Ahhh,{w=0.2} {nw}"
     extend 5dkbfa "just the thought of it makes me feel all tingly, [player]~"
     return
+    
+    
+    init 5 python:
```
Comment (Booplicate) [inline Monika After Story/game/script-topics.rpy:14272]:
```suggestion
init 5 python:
```

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -14267,3 +14267,33 @@ label monika_hot_springs:
...
+        Event(
+            persistent.event_database,
+            eventlabel='monika_languagenuances',
+            prompt="Language nuances",
+            category=['literature', 'trivia'],
+            random=True
+        )
+    )
+
+label monika_languagenuances:
```
Comment (Booplicate) [inline Monika After Story/game/script-topics.rpy:14283]: For readability
```suggestion
label monika_language_nuances:
```

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -14267,3 +14267,33 @@ label monika_hot_springs:
     m 5dkbfb "Ahhh,{w=0.2} {nw}"
     extend 5dkbfa "just the thought of it makes me feel all tingly, [player]~"
     return
+    
+    
+    init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel='monika_languagenuances',
```
Comment (Booplicate) [inline Monika After Story/game/script-topics.rpy:14276]:
```suggestion
            eventlabel='monika_language_nuances',
```

**Pull request merged**